src/state/apis/snapshot/snapshot.ts

```diff
@@ -165,23 +164,25 @@ export const snapshotApi = createApi({
             FEE_CURVE_PARAMETERS[model].FEE_CURVE_FOX_DISCOUNT_DELAY_HOURS,
           )
           const delegation = false // don't let people delegate for discounts - ambiguous in spec
-          const votingPowerResults = await Promise.allSettled(
-            evmAddresses.map(async address => {
-              const votingPowerUnvalidated = await getVotingPower(
-                address,
-                '1',
-                strategies,
-                foxDiscountBlock,
-                SNAPSHOT_SPACE,
-                delegation,
-              )
-              // vp is FOX in crypto balance
-              return bnOrZero(VotingPowerSchema.parse(votingPowerUnvalidated).vp)
-            }),
+
+          const votingPowerResults = await getVotingPower(
+            evmAddresses,
+            '1',
+            strategies,
+            foxDiscountBlock,
+            SNAPSHOT_SPACE,
+            delegation,
           )
-          const foxHeld = BigNumber.sum(
-            ...votingPowerResults.filter(isFulfilled).map(r => r.value),
-          ).toNumber()
+
+          const scores = ScoresSchema.parse(votingPowerResults).scores
+
+          const foxHeld = scores.reduce((acc: number, scoreByAddress: Record<string, number>) => {
+            const values = Object.values(scoreByAddress)
+
+            if (!values.length) return acc
+
+            return acc + BigNumber.sum(...values.map(bnOrZero)).toNumber()
+          }, 0)
```

Comment:
Non-blocking: See previous comment regarding `Number.MAX_SAFE_INTEGER`

Reply:
Fixed in [0275bec](https://github.com/shapeshift/web/pull/8057/commits/0275becb4da42956c26e33f854a40e300206058a)